### DISCUSSION TITLE
How to get Host name/IP for transactions

### DISCUSSION THREAD

**Discussion by AjinThankachan** [apache/skywalking#10241] [Q&A]
Hi,
OAP : 8.9

I am having OAP 8.9 in my env and i am monitoring java applications. Is there any possibility that i can differentiate the transactions based on the IP or host name or the user who hit the application? Thanks in advance.

**Comment by wu-sheng** [comment]
Why is this a question for SkyWalking? This seems irrelevant.

**Reply by AjinThankachan** [reply]
Hi @wu-sheng since it's Skywalking is an APM tool so we are testing the application with load runner tools.  And we need to segregate the data based on the load testing tool ip or details or which user or session accessed the site.To do the testing.  So i was checking is there any functionality likewise to get the information in Skywalking backend or in the interface. 
Also the  RUM functionality is available in Skywalking?

**Reply by wu-sheng** [reply]
I think we have provided browser monitoring for now, but it doesn't gather user IP or session information. 
https://github.com/apache/skywalking-client-js
